tests/providers/databricks/operators/test_databricks_sql.py

```diff
@@ -22,10 +22,10 @@
 from unittest.mock import patch
 
 import pytest
-from databricks.sql.types import Row
```

Comment:
Why change this import?

Reply:
Sorry. have not seen it before merging. It's a left-over from some other trials done - not a big harm, it's the same Row just imported from `databricks_sql` - so it actually might even make more sense to be imported from there.